### PR TITLE
Added stereo launch file and enabled roslaunch checks for all launch…

### PR DESCRIPTION
**Pull request by tonybaltovski**
… files.  Also, used parameter for camera name which will be the namespace.

### CODE REVIEW

**Comment by tonybaltovski** [comment]
Failed cause of #64.

File: pointgrey_camera_driver/launch/stereo.launch [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+  <arg name="left_camera_serial" default="0" />
+  <arg name="left_camera_calibrated" default="0" />
+
+  <arg name="right_camera_serial" default="0" />
+  <arg name="right_camera_calibrated" default="0" />
+
+  <group ns="$(arg camera_name)" >
+    <group ns="left" >
+      <node pkg="nodelet" type="nodelet" name="left_camera_nodelet_manager" args="manager" />
```
Comment (mikepurvis) [inline pointgrey_camera_driver/launch/stereo.launch]: Couldn't both cameras go in the same nodelet manager, or does that cause namespacing chaos?

Comment (tonybaltovski) [inline pointgrey_camera_driver/launch/stereo.launch]: It is namespacing chaos.

Comment (mikepurvis) [inline pointgrey_camera_driver/launch/stereo.launch]: Maybe add a comment explaining that? Otherwise LGTM.

**Comment by tonybaltovski** [comment]
I will switch to the stereo nodelet which will allow both cameras to be loaded into one nodelet manager.

**Pull request closed without merging**